lib/ansible/modules/database/postgresql/postgresql_tablespace.py

```diff
@@ -248,16 +248,15 @@ def get_info(self):
             query = ("SELECT r.rolname, (SELECT Null), %s "
                      "FROM pg_catalog.pg_tablespace AS t "
                      "JOIN pg_catalog.pg_roles AS r "
-                     "ON t.spcowner = r.oid "
-                     "WHERE t.spcname = '%s'" % (location, self.name))
+                     "ON t.spcowner = r.oid " % location)
         else:
             query = ("SELECT r.rolname, t.spcoptions, %s "
                      "FROM pg_catalog.pg_tablespace AS t "
                      "JOIN pg_catalog.pg_roles AS r "
-                     "ON t.spcowner = r.oid "
-                     "WHERE t.spcname = '%s'" % (location, self.name))
+                     "ON t.spcowner = r.oid " % location)
 
-        res = exec_sql(self, query, add_to_executed=False)
+        res = exec_sql(self, query + "WHERE t.spcname = %(name)s",
```

Comment:
Just curious why you're using the where condition here and not in the sql statement above?

Reply:
as i remember, it doesn't work if you have %s and %(blabla)s in the same row in one instruction

Reply:
```
Python 3.6.6 (default, Mar 29 2019, 00:03:27) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-36)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> s = "%s %(blabla)s" % 'first'
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: format requires a mapping
```